api/repositories/service_instance_repository_test.go

```diff
@@ -1052,13 +1073,14 @@ var _ = Describe("ServiceInstanceRepository", func() {
 		It("purges the service instance", func() {
```

Comment:
This test should check that both the instance and the binding are not found in an `Eventually` block. Removing the finalizer is an implementation detail.

Reply:
@georgethebeatle The service binding wont be deleted since the with envtest we do not have the garbage collector controller. As per: https://book.kubebuilder.io/reference/envtest#testing-considerations

Reply:
> @georgethebeatle The service binding wont be deleted since the with envtest we do not have the garbage collector controller. As per: [book.kubebuilder.io/reference/envtest#testing-considerations](https://book.kubebuilder.io/reference/envtest#testing-considerations)

makes sense, indeed